sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/api/Azure.ResourceManager.DesktopVirtualization.netstandard2.0.cs

```diff
@@ -945,6 +1181,24 @@ public partial class MsixPackagePatch : Azure.ResourceManager.Models.ResourceDat
         public override string ToString() { throw null; }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct PublicNetworkAccess : System.IEquatable<Azure.ResourceManager.DesktopVirtualization.Models.PublicNetworkAccess>
```

Comment:
This name is quite generic, could we rename this to `DesktopVirtualizationPublicNetworkAccess`?